book/src/design/proving-system/circuit-commitments.md

```diff
@@ -4,14 +4,14 @@
 
 At the start of proof creation, the prover has a table of cell assignments that it claims
 satisfy the constraint system. The table has $n = 2^k$ rows, and is broken into advice,
-auxiliary, and fixed columns. We define $F_{i,j}$ as the assignment in the $j$th row of
+instance, and fixed columns. We define $F_{i,j}$ as the assignment in the $j$th row of
 the $i$th fixed column. Without loss of generality, we'll similarly define $A_{i,j}$ to
-represent the advice and auxiliary assignments.
+represent the advice and instance assignments.
 
 > We separate fixed columns here because they are provided by the verifier, whereas the
-> advice and auxiliary columns are provided by the prover. In practice, the commitments to
-> auxiliary and fixed columns are computed by both the prover and verifier, and only the
-> advice commitments are stored in the proof.
+> advice columns are provided by the prover. The instance columns are agreed between both
+> parties. In practice, the commitments to instance and fixed columns are computed by
```

Comment:
This change makes the point I was trying to convey less clear. I want the reader to understand that the reason we defined a separate `F_{i,j}` but not a separate `I_{i,j}` is because both advice and instance columns are only defined at proving time (and within the proving system are treated almost identically), whereas fixed columns are defined in advance (hence "provided by the verifier"). It does not matter that both parties agree on the instance columns; in practice, the prover provides them to the verifier out-of-band (wrt the proof), and any rejection of the instance column contents occurs before the verifier participates in the verification protocol.